src/main/java/com/wultra/security/userdatastore/userclaims/UserClaimsController.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * User Data Store
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.security.userdatastore.userclaims;
+
+import io.swagger.v3.oas.annotations.Operation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+/**
+ * REST controller providing API for CRUD of user claims.
+ *
+ * @author Lubos Racansky lubos.racansky@wultra.com
+ */
+@RestController
+class UserClaimsController {
+
+    private static final Logger logger = LoggerFactory.getLogger(UserClaimsController.class);
+
+    private final UserClaimsService userClaimsService;
+
+    UserClaimsController(UserClaimsService userClaimsService) {
```

Comment:
Package private constructor?

Reply:
Is there any problem with that? In general, I would like to use fewer _public_ access modifiers, and this works for Spring.

Reply:
OK